src/battle_ai_main.c

```diff
@@ -3721,8 +3731,9 @@ static s16 AI_CheckViability(u8 battlerAtk, u8 battlerDef, u16 move, s16 score)
         break;
     case EFFECT_WISH:
     case EFFECT_HEAL_BELL:
+    case EFFECT_TAKE_HEART:
```

Comment:
This effect is more like Refresh than Heal Bell, and probably shouldn't be using a function for Aromatherapy.

Reply:
Tbh, I took the lazy route and didn't check much.

The function even has code specific to Wish and Aromatherapy, so it was a terrible idea to use it 😂 

I'll just stack it with the code for Refresh's effect and call it a day.